src/main/resources/application.properties

```diff
@@ -21,6 +21,11 @@ server.port=8080
 logging.level.root=${DEFAULT_LOGGING_LEVEL:INFO}
 logging.level.it.gov.pagopa=${APP_LOGGING_LEVEL:INFO}
 logging.level.com.zaxxer.hikari=${APP_LOGGING_LEVEL:INFO}
+# to show sql
+spring.jpa.show-sql=${SHOW_SQL:true}
```

Comment:
is it correct that **_show SQL_** is true by default if no parameter is defined? Can be a vulnerability in UAT/PROD environment if the parameter is not defined, maybe.

Reply:
done